api/server.go

```diff
@@ -52,12 +53,17 @@ type TsuruHandler struct {
 	h       http.Handler
 }
 
+type serviceManager struct {
+	Team authTypes.TeamService
+}
+
 func fatal(err error) {
 	fmt.Println(err.Error())
 	log.Fatal(err.Error())
 }
 
 var tsuruHandlerList []TsuruHandler
+var ServiceManager *serviceManager
```

Comment:
Why this global var instead of using servicemanager package like in others packages?

Reply:
Thanks, I forgot to remove it.